install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -986,10 +986,11 @@ data:
 {{- end }}
 {{- end }}
 {{- if .Values.hubble.export }}
-  hubble-export-file-max-size-mb: {{ .Values.hubble.export.fileMaxSizeMb | quote }}
-  hubble-export-file-max-backups: {{ .Values.hubble.export.fileMaxBackups | quote }}
-  hubble-export-file-compress: {{ .Values.hubble.export.fileCompress | quote }}
 {{- if .Values.hubble.export.static.enabled }}
+  {{- /* hubble export configurations moved, use default to make upgrades seemless */ -}}
```

Comment:
Similar as above, we should add a TODO statement here to eventually remove the fallback on the old values.

Reply:
Yes will do